src/scripts/alleles/G_domain.py

```diff
@@ -0,0 +1,197 @@
+from Bio import SeqIO, BiopythonWarning, BiopythonParserWarning
+import csv
+import sys
+import os
+sys.path.append(os.getcwd())
+from io import StringIO
+import warnings
+import logging
+
+EXCLUDED_GENES = {
+    "BTN3A",
+    "CD1",
+    "DOA",
+    "DOB",
+    "V",
+    "P",
+    "S",
+    "Y",
+    "MICA",
+    "MICB",
+    "MR1",
+    "HFE",
+    "J",
+    "L",
+    "TAP1",
+    "DMB",
+    "DMA",
+    "T",
+    "V",
+    "W",
+    "K",
+    "TAP2",
+    "U",
+    "H",
+    "DRB8",
+    "DRB9",
+    "DRB7",
+    "DRB6",
+    "DPB2",
+    "DPA2",
+    "DQA2",
+    "DRB2"
+}
+
+#logging.basicConfig(filename = 'build/biopython.log', filemode = 'w', level = logging.DEBUG )
+#logging.basicConfig(filename = 'biopython.log', filemode = 'w', level = logging.INFO )
+logging.basicConfig(filename = 'biopython.log', filemode = 'w', level = logging.WARNING )
```

Comment:
This still needs to go in the `build/` directory. Further down, you open the log `build/biopython.log`, which causes a `FileNotFoundError`.